currency_rates/tests.py

```diff
@@ -170,10 +171,10 @@ def test_load_rates_ignore_non_existent(self):
         # create currency
         Currency.objects.create(code="USD")
 
-        with mock.patch('urllib2.urlopen') as mock_urlopen:
+        with mock.patch('currency_rates.management.commands.load_rates.urlopen') as mock_urlopen:
```

Comment:
dunno, mocking six doesn't work here (probably happened too late?) so i gave up and did this instead